### PR TITLE
suggest two strategies for conflict resolution

### PR DESCRIPTION
**Pull request by rosenhouse**
Reading the docs for the first time, I saw this section could use a bit of help.

Change is suggesting a couple different approaches to this problem.

This is my first PR to anything Kubernetes-related in a long time.  Any feedback on content or process is welcome.

Thanks.

### CODE REVIEW

**Comment by bowei** [comment]
We did some work on the mkdocs to pin its version -- can you try regenerating and re-pushing

File: docs-src/concepts.md [hunk 1]
```diff
@@ -106,7 +106,12 @@ configuration. For example, two application teams may inadvertently submit
 configuration for the same HTTP path. There are several different strategies for
 handling this:
 
-* TODO
+* A controller may detect this situation during reconciliation and choose to honor
+  one resource or other, perhaps based on a timestamp (e.g. first write wins).
+  Resource `status` fields would reflect the result.
+* A [validating admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
```
Comment (bowei) [inline docs-src/concepts.md]: Is it possible for a webhook to know state globally? I think if you have replicated API servers, you won't see all requests at all webhooks (imagine having a webhook side by side on `localhost` to the API server).

Comment (bowei) [reply docs-src/concepts.md]: (Maybe this is just  a caveat)

File: docs-src/concepts.md [hunk 2]
```diff
@@ -106,7 +106,12 @@ configuration. For example, two application teams may inadvertently submit
 configuration for the same HTTP path. There are several different strategies for
 handling this:
 
-* TODO
+* A controller may detect this situation during reconciliation and choose to honor
+  one resource or other, perhaps based on a timestamp (e.g. first write wins).
```
Comment (bowei) [inline docs-src/concepts.md]: Do Kubernetes objects always come with modification timestamps?
I know that creation time is standard.

Comment (hbagdi) [reply docs-src/concepts.md]: They don't have a modification timestamp at all. They is resourceVersion and generation but there is no way to figure out which was the last updated one.

In our controller, whenever such issues arise, the controller respects the resource which has the oldest creation time.

**Comment by robscott** [comment]
Hey @rosenhouse, thanks for your work on this! It looks like this PR has gone stale so I'm going to close it for now. We would still love help as far as conflict resolution though. We've been working through a variety of different conflicts and how we should resolve them [in a doc](https://docs.google.com/document/d/1ArLzUF_oJr3jV-fhXKfaEBsvRXW-6p6UuaB9FGuosIg/edit) and gradually turning that into PRs, with https://github.com/kubernetes-sigs/service-apis/pull/263 as the first one. Feel free to contribute to that doc, follow up on Slack, or reopen this PR if you're still interested in this.

/close

**Pull request closed without merging**